### ISSUE TITLE
import problem

### ISSUE THREAD

**Issue by BramDo** [Strilanc/PaperImpl-2017-DirtyPeriodFinding#1]
I already have an install of projectq, and this works fine. However, in this project I got a ModuleNotFoundError : No module named 'decompositions'. Is it something with the __init__.py in the src directory

Update
After changing the __init__.py to a empty file it works.

**Comment by Strilanc** [comment]
Do you mean you added an `__init__.py` file to the `src/` directory, and this fixes an error you had when trying to run the code?

Could you post exactly which commands you were running?

**Comment by BramDo** [comment]
I did not add an  __init__.py file to the src/ directory. I changed the  __init__.py file in the src/dirty_period_finding  sub directory and I made this file an empty file.
I run the command python src/factor.py 55 in the paperImpl-2017....... directory and this works.
The command pytest collected 138 items with only 2 errors, one in the bootstrap_ancilla_test.

I read the blog post and parts of the paper but would like to have more insight in this great project, the structure of the software and also the test files. The ascii circuit test looks good.
